ros2lifecycle/ros2lifecycle/verb/set.py

```diff
@@ -44,7 +44,12 @@ def main(self, *, args):  # noqa: D102
             node_names = get_node_names(
                 node=node, include_hidden_nodes=args.include_hidden_nodes)
 
-        if args.node_name not in node_names:
+        node_found = False
+        for node_name in node_names:
+            if args.node_name in node_name.full_name:
+                node_found = True
+                break
+        if not node_found:
```

Comment:
This could be much shorter: `if args.node_name not in {n.full_name for n in node_names}:`

Reply:
of course there is a one-liner for python :) thanks!